typescript/sdk/src/ism/read.ts

```diff
@@ -184,9 +192,27 @@ export class EvmIsmReader implements IsmReader {
 
   async deriveNullConfig(
     address: Address,
-  ): Promise<
-    WithAddress<PausableIsmConfig | TestIsmConfig | OpStackIsmConfig>
-  > {
+  ): Promise<WithAddress<NullIsmConfig>> {
+    // if it has trustedRelayer() property --> TrustedRelayer ISM
```

Comment:
thanks for adding! might need to also mock `TrustedRelayerIsm__factory` in the associated test to make CI happy